CONTRIBUTING.md

```diff
@@ -66,20 +66,25 @@ We love pull requests. Here's a quick guide:
 
 1. Fork the repo.
 
-2. Run the tests. We only take pull requests with passing tests, and it's great
+2. Ensure you have the development requirements:
+
+   * node & npm
```

Comment:
@stefanpenner do you know the minimum version requirements for each dependency off hand? Happy to update the PR.